scripts/README.md

```diff
@@ -0,0 +1,22 @@
+<div style="text-align:center">
+    <h2>Sidecar Scripts</h2>
+    <div>
+    A set of notes and instructions for scripts used in Substrate-api-sidecar
+    </div>
+</div>
+<br></br>
+
+
+## Script `run_chain_tests.py`
+
+### Summary
+
+This script calls the local runtime-tests helper library in order to test the current branch or development enviornment against 
+a collection of different blocks, across different runtimes. It does this for Polkadot, Kusama, and Westend. 
+
+### Requirements
```

Comment:
It would be better to do:

To save the file `requirements.txt` by `$ pip freeze > requirements.txt`
Then just state `$ pip install -r requirements.txt` in the README instead

Reply:
Awesome! I was hesitant at first to put a `requirements.txt` for one dependency, but this convinces me, Thanks!

Reply:
Awesome fixed.